scss/_variables.scss

```diff
@@ -1184,7 +1165,7 @@ $form-color-disabled-filter:            brightness(0) invert(1) brightness(.8) !
 
 // scss-docs-start form-check-variables
 $form-check-input-width:                  1em !default;
-$form-check-min-height:                   $font-size-base * $input-btn-line-height !default;
+$form-check-min-height:                   calc(#{$font-size-base} * #{$input-btn-line-height}) !default; // stylelint-disable-line function-disallowed-list
```

Comment:
Should we explain the fact that some Sass variables like `$font-size-base`  now reference a CSS var and so should be used in calculations with `calc`?